src/components/newResopage/NavbarTopLoggedIn.jsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Disclosure,
+  DisclosureButton,
+  DisclosurePanel,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuItems,
+} from "@headlessui/react";
+import { Bars3Icon, BellIcon, XMarkIcon } from "@heroicons/react/24/outline";
+
+const navigation = [
+  { name: "Daily Quests", href: "#", current: false },
+  { name: "My Goals", href: "#", current: false },
+  { name: "New Resolutions", href: "#", current: false },
+  { name: "Calendar", href: "#", current: false },
+];
+
+function classNames(...classes) {

```

Comment:
This won't normally work with tailwind. For tailwind there is a tailwind-merge package available in mpm